tests/unit/tasks/serve-test.js

```diff
@@ -70,7 +70,7 @@ describe('serve task', function() {
     });
 
     it(`Serves ember app from given path`, function() {
-      runServeTask('assets');
+      runServeTask('docs');
```

Comment:
can you explain this change?

Reply:
In this PR I've removed the `assets` directory.

The `serve` task may take `path` as an option. It is the build directory to serve.

This test is covering this specific option. As I deleted `assets`, I had to replace it with another existing directory.

Reply:
ahhh, I see, thanks!

Reply:
ahah this failing test wasn't clear to me as well :)